### PR TITLE
Mobile,ColoredListView: Improve widget documentation readability

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem] Code example contained mixed HTML and JS code
[Solution] Separate JS and HTML code in examples

Signed-off-by: Pawel Kaczmarczyk <p.kaczmarczy@samsung.com>

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/601

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_ColoredListview.htm [hunk 1]
```diff
@@ -38,10 +38,11 @@ <h2><a id="manual-constructor"></a>Manual Constructor</h2>
    &lt;li class=&quot;ui-li-static&quot;&gt;Item&lt;/li&gt;
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_ColoredListview.htm]: Suggestion. How about adding "\<p>HTML code:\</p>" before line 35?

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_ColoredListview.htm]: Done.

**Comment by TomaszLukawskiSam** [comment]
Looks good to me. Thank you.

**Pull request merged**